js/data.js

```diff
@@ -0,0 +1,56 @@
+import {getRandomArrayElement} from './utils.js';
+import {getRandomInt} from './utils.js';
```

Comment:
```suggestion
import {getRandomArrayElement, getRandomInt} from './utils.js';
```
с одного файла все импортируй разом